packages/dialogs/src/RPA/Dialogs/library.py

```diff
@@ -821,6 +821,48 @@ def add_checkbox(
 
         self.add_element(element)
 
+    @keyword("Add dialog next page button")
```

Comment:
It is unnecessary declare keyword name in the decorator as name is the same as method name.

Reply:
ah, didn't know that, thanks 😄 I went based on examples and all the keywords in RPA.Dialogs also had the keyword name in the decorator. Removed